CONTRIBUTING.md

```diff
@@ -79,43 +108,37 @@ submit a pull request:
    usage presented as doctests when appropriate.
 
 -  Please prefix the title of your pull request with `[MRG]` (Ready for Merge),
-   if the contribution is complete and ready for a detailed review. Two core
-   developers will review your code and change the prefix of the pull request to
-   `[MRG + 1]` on approval, making it eligible for merging. An incomplete
-   contribution -- where you expect to do more work before receiving a full
-   review -- should be prefixed `[WIP]` (to indicate a work in progress) and
+   if the contribution is complete and ready for a detailed review. Some of the
+   core developers will review your code, make suggestions for changes, and
+   approve it as soon as it is ready for merge. Pull requests are usually merged
+   after two approvals by core developers, or other developers asked to review the code. 
+   An incomplete contribution -- where you expect to do more work before receiving a full
+   review -- should be prefixed with `[WIP]` (to indicate a work in progress) and
    changed to `[MRG]` when it matures. WIPs may be useful to: indicate you are
    working on something to avoid duplicated work, request broad review of
    functionality or API, or seek collaborators. WIPs often benefit from the
    inclusion of a
    [task list](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments)
    in the PR description.
 
--  When adding additional functionality, provide at least one
-   example script in the ``examples/`` folder. Have a look at other
+-  When adding additional functionality, check if it makes sense to add one or
+   more example scripts in the ``examples/`` folder. Have a look at other
    examples for reference. Examples should demonstrate why the new
    functionality is useful in practice and, if possible, compare it
    to other methods available in pydicom.
 
 -  Documentation and high-coverage tests are necessary for enhancements to be
-   accepted. Bug-fixes or new features should be provided with 
-   [non-regression tests](https://en.wikipedia.org/wiki/Non-regression_testing).
-   These tests verify the correct behavior of the fix or feature. In this
-   manner, further modifications on the code base are granted to be consistent
-   with the desired behavior.
-   For the Bug-fixes case, at the time of the PR, this tests should fail for
-   the code base in master and pass for the PR code.
-
--  The documentation should also include expected time and space
-   complexity of the algorithm and scalability, e.g. "this algorithm
-   can scale to a large number of samples > 100000, but does not
-   scale in dimensionality: n_features is expected to be lower than
-   100".
+   accepted. Bug-fixes shall be provided with 
+   [regression tests](https://en.wikipedia.org/wiki/regression_testing) that
+   fail before the fix, for new features the correct behavior shall be verified
```

Comment:
comma splice

Reply:
Ok, had to look this up... learning something new every day :)